### PR TITLE
Translation to Turkish

### LINKED ISSUE #6158: Turkish translate for main md files needed
### Topic

I want to make it available for Turkish people who wants to access.

### PR DESCRIPTION
**Pull request by neondev27**
Resolves #6158

 Changes:
Main READMEs are updated to contain Turkish translate access. 

 Screenshots of the change:
pull request

pr2
pr3
pr4

#### PR Checklist

- [x] Main markdown files translated in their new copies named in Turkish language

### CODE REVIEW

File: README.md
Comment (osbm) [inline README.md:1]: This link should not redirect to your own fork

File: KATKIDABULUNMA.md
Comment (osbm) [inline KATKIDABULUNMA.md:1]: You should change the filename, filenames would be understandable to English speakers
For example like this:
CONTRIBUTING.tr.md

File: DAVRANIS_KODU.md
Comment (osbm) [inline DAVRANIS_KODU.md:1]: You should change the filename, filenames should be understandable to English speakers
For example like this:
CODE_OF_CONDUCT.tr.md

File: BENIOKU.md
Comment (osbm) [inline BENIOKU.md:1]: You should change the filename, filenames should be understandable to English speakers
For example like this:
README.tr.md

**Review by osbm — CHANGES_REQUESTED** [review]
Other than my comments on these files this is a very good translation. Thank you :)

**Comment by neondev27** [comment]
Done!

**Comment by Qianqianye** [comment]
Thanks @neondev27 for translation the README.md file to Turkish and thanks @osbm for reviewing it. As I mentioned in the [comment](https://github.com/processing/p5.js/issues/6158#issuecomment-1584008039), we are rolling out new languages slowly to be absolutely sure that we can support the full translation and maintenance of each language. We'd like to come up with a system to manage different translation before we take in more translations as discussed in issue#[5208](https://github.com/processing/p5.js/issues/5208). Before the system is finalized, you can host the Turkish version of the main README.md and CODE_OF_CONDUCT_TR.md you translated on an external blog or site. And we can include the link to the external site on p5.js repos.

We do have an open issue#[4137](https://github.com/processing/p5.js/issues/4137) discussing the translation of the contributor docs. You can add a Turkish folder in the [contributor_docs](https://github.com/processing/p5.js/tree/main/contributor_docs) folder and add a Turkish translation of the this [README.md](https://github.com/processing/p5.js/tree/main/contributor_docs#readme) file, then we can add the Turkish tab on the [top nav bar](https://p5js.org/contributor-docs/#/).

I will close this PR for now. Please feel free to open a new PR if you'd like to add Turkish translation of the contributor docs. Thank you!

**Pull request closed without merging**